src/com/tigersale/userInterface/inventoryManager/EditProductView.java

```diff
@@ -150,4 +182,24 @@ private void editAmount(Product product) {
             }
         }
     }
+
+    private void editBrand(Product product) {
```

Comment:
The database should be updated at line 129 `result = ProductTable.updateProductValues(product);` or does that not actually update the database?